packages/entities/entities-plugins/src/components/PluginForm.vue

```diff
@@ -1335,6 +1335,16 @@ const getRequestBody = computed((): Record<string, any> => {
   return requestBody
 })
 
+// Read-only record for the View Configuration feature
+const viewConfigurationRecord = computed(() => {
+  const record = { ...getRequestBody.value }
+
+  // Don't show UI data in the configuration view
+  delete record['__ui_data']
```

Comment:
```suggestion
  // eslint-disable-next-line @typescript-eslint/no-unused-vars
  const { __ui_data, ...record } = getRequestBody.value
```

I think we can skip delete and use destructuring instead.